lib/index.ts

```diff
@@ -4,18 +4,20 @@ export function setupXStateTest<TestContext, TContext>(
   hooks: NestedHooks,
   testModel: TestModel<TestContext, TContext>
 ) {
-  hooks.after(async function(assert) {
+  hooks.after(function(assert) {
     try {
-      await testModel.testCoverage();
+      testModel.testCoverage();
```

Comment:
I'm confused the docs specifically state the use of `await` in this case.

Reply:
@davidkpiano ?

Is the type wrong on here?

Reply:
according to https://xstate.js.org/docs/packages/xstate-test/#api it doesn't need to be awaited?